src/calculateRentalCost.js

```diff
@@ -27,8 +27,18 @@
  * @return {number}
  */
 
+const COST_ONE_DAY = 40;
+
 function calculateRentalCost(days) {
-  // write code here
+  const cost = days * COST_ONE_DAY;
+
+  if (days >= 7) {
+    return cost - 50;
+  } else if (days >= 3) {
+    return cost - 20;
+  }
```

Comment:
please create variables for all custom numbers - [checklist ](https://github.com/mate-academy/js_task-transportation-on-vacation/blob/master/checklist.md) point 2